pkg/vault/test/foundry/VaultLiquidityRate.t.sol

```diff
@@ -213,4 +213,25 @@ contract VaultLiquidityWithRatesTest is BaseVaultTest {
             bytes("")
         );
     }
+
+    function testRemoveLiquiditySingleTokenExactOutWithRate__Fuzz(uint256 wstEthRate, uint256 wstEthAmountOut) public {
```

Comment:
This test passes either way, but doesn't hurt to add. We'd need to have a Vault mock that overrode and mutated the amountsIn to "really" test this, I guess, but that seems like overkill for a simple thing.

Reply:
Actually it can be done in existing tests: https://github.com/balancer/balancer-v3-monorepo/pull/946/commits/87de278d4a8844b5e099bb80dae61bd3ab60cd6a.

It's an overkill, but I'd say it's worth it. Thanks for the idea.